components/Accordion.js

```diff
@@ -0,0 +1,39 @@
+
+import Chevron from "./Chevron.js";
+import React, { useState, useRef } from 'react';
+import styles from "../styles/Accordion.module.css";
+
+export default function Accordion({accordionTitle}, {accordionContent}) {
+  const [Active, setActiveState] = useState(false);
+  const [setHeight, setHeightState] = useState("0px");
+  const [setRotate, setRotateState] = useState("accordion__icon");
+
```

Comment:
Can you change your state names to be more consistent? Generally the pattern is "stateName" then "setStateName" in camel case

Active -> isActive, setisActive. (isActive since its a boolean, active and setActive would be alright too)
setHeight -> height, setHeight
setRotate -> rotate, setRotate